src/commands/user-commands/tutor-log.ts

```diff
@@ -0,0 +1,103 @@
+import { Command } from "../../interfaces/Command";
+import { commandLog, sendError } from "../../utils/logs";
+import { SlashCommandBuilder } from "discord.js";
+import { reportOptions } from "../../utils/options";
+import { TutorLogInsert } from "../../utils/types";
+import { getContactId, getTutorId, insertTutorLog } from "../../utils/supabase";
+import { createEmbeded } from "../../utils/embeded";
+
+
+export const tutorlog: Command = {
+  
+    data : new  SlashCommandBuilder()
+      .setName("tutor-log")
+      .setDescription("Log your tutor hours")
+      .addStringOption((option) =>
+      option
+      .setName("tutoring-type")
+      .setDescription("Online or in-person?")
```

Comment:
Keep option descriptions as an actual descriptor of what the option means rather than what the options actually are. Something like "Type of tutoring session you engaged in!" should be good.